samples/iothub-connect-to-eventhubs/IotHubConnection.cs

```diff
@@ -0,0 +1,283 @@
+﻿
```

Comment:
The things in this area are not part of this set of changes;  they're covered under another PR and I'll rebase them out when that closes.